tests/test_whoas.py

```diff
@@ -0,0 +1,31 @@
+import transmogrifier.models as timdex
+from transmogrifier.helpers import parse_xml_records
+from transmogrifier.sources.whoas import Whoas
+
+
+def test_whoas_get_content_type_returns_accepted_content_type():
```

Comment:
For clarity of testing logic, the second test should:

1. Confirm that the length of `input_records` is 2 and the length of `output_records` is 1 to verify that we started with 2 records and 1 was skipped.
2. Checking for the `content_type` value of the returned record is fine, but checking the entire contents of the record seems unnecessary and adds confusion about what is being tested.

Might as well update the test names too, to match the actual logic of what we're testing now.